aws-lookoutvision-project/resource-role.yaml

```diff
@@ -15,6 +15,13 @@ Resources:
             Principal:
               Service: resources.cloudformation.amazonaws.com
             Action: sts:AssumeRole
+            Condition:
```

Comment:
This will likely break publishing. `test-type` fails if you don't manually remove the conditions.

Reply:
This code is auto-generated from `cfn generate` we would want to figure that out pretty quickly with the CloudFormation team if that's the case.

Reply:
Removed